cartography/intel/aws/emr.py

```diff
@@ -45,11 +43,9 @@ def get_emr_describe_cluster(boto3_session: boto3.session.Session, region: str,
         response = client.describe_cluster(ClusterId=cluster_id)
         cluster_details = response['Cluster']
     except botocore.exceptions.ClientError as e:
-        logger.warning(
-            "Could not run EMR describe_cluster due to boto3 error %s: %s. Skipping.",
-            e.response['Error']['Code'],
-            e.response['Error']['Message'],
-        )
+        code = e.response['Error']['Code']
+        msg = e.response['Error']['Message']
+        logger.warning(f"Could not run EMR describe_cluster due to boto3 error {code}: {msg}. Skipping.")
```

Comment:
Figured I'd sneak in a change here to make the string formatting more readable since we expect new module authors to be copying this